src/platform/intel/cavs/platform.c

```diff
@@ -341,6 +341,7 @@ int platform_init(struct sof *sof)
 	platform_timer_start(platform_timer);
 
 	trace_point(TRACE_BOOT_PLATFORM_CLOCK);
+	platform_clock_init();
```

Comment:
Can't `platform_clock_init` be called from within `clock_init` on each platform?